src/scaffoldfitter/fitter.py

```diff
@@ -825,6 +829,28 @@ def getDataRMSAndMaximumProjectionError(self):
             del error
         return rmsErrorValue if (rmsResult == RESULT_OK) else None, maxErrorValue if (maxResult == RESULT_OK) else None
 
+    def getDataRMSAndMaximumProjectionErrorForGroup(self, groupName):
+        """
+        Get RMS and maximum error for the nodeset group with the given name.
+        If the groupName is not a valid nodeset group name then -1, -1 is returned.
+
+        :param groupName: Name of group to make calculation over.
+        :return: RMS error value, maximum error value.
+        """
+        with ChangeManager(self._fieldmodule):
+            group = self._fieldmodule.findFieldByName(groupName).castGroup()
+            if group.isValid():
+                calculation_field  = self._fieldmodule.createFieldAnd(group, self._activeDataGroupField)
+                nodeset = self._fieldmodule.findNodesetByFieldDomainType(Field.DOMAIN_TYPE_DATAPOINTS)
+                temp_dataset_group = self._fieldmodule.createFieldGroup().createNodesetGroup(nodeset)
+                temp_dataset_group.addNodesConditional(calculation_field)
+                result = self.getDataRMSAndMaximumProjectionError(temp_dataset_group)
+                del temp_dataset_group
+                del calculation_field
+                return result
+
+        return -1, -1
```

Comment:
Should return None, None for consistency with getDataRMSAndMaximumProjectionError(). Also update docstring comment.

Reply:
I intentionally returned -1, -1 so the caller would know that the group was invalid and that the computation failed.